string name
int32 value
---
industrial_msgs/ServiceReturnCode code